packages/netlify-cms-core/src/actions/editorialWorkflow.ts

```diff
@@ -173,7 +163,6 @@ function unpublishedEntryStatusChangeRequest(
       oldStatus,
```

Comment:
Can we remove `oldStatus` and `newStatus` from the payload?
See https://github.com/netlify/netlify-cms/pull/4289/files#diff-c04b7dd92be649549b8b8806542af69cR112